server/src/metadata.rs

```diff
@@ -148,18 +149,63 @@ pub mod entities_metadata_query {
             .await?
             .into_iter()
             .map(|result| {
+                let identifier = result.id as i32;
                 let title: Option<String> = result.params.as_ref()
                     .and_then(|params| params.get("title"))
                     .and_then(|title| title.as_str())
                     .map(|title| title.to_string());
                 let id = get_iri(result.id as i32);
                 let schema_type =
                         get_schema_type(&result.resource_type);
-                let name = title.clone().unwrap_or_else(|| format!("{schema_type}: {id}"));
+                let name = title.clone().unwrap_or_else(|| {
+                    let schema_type_i18n = match result.instance.as_str() {
+                        "de" => {
+                            match result.resource_type.as_str() {
+                                "article" => "Artikel",
+                                "course" => "Kurs",
+                                "text-exercise" => "Aufgabe",
+                                "text-exercise-group" => "Aufgabengruppe",
+                                "video" => "Video",
+                                "applet" => "Applet",
+                                _ => "Inhalt",
+                            }
+                        },
+                        _ => {
+                            match result.resource_type.as_str() {
+                                "article" => "Article",
```

Comment:
Do we have to capitalize them?

Reply:
Yep, those words are used at the beginning of `name`